flink-libraries/flink-table/src/main/scala/org/apache/flink/table/functions/sql/ScalarSqlFunctions.scala

```diff
@@ -58,4 +58,22 @@ object ScalarSqlFunctions {
       OperandTypes.family(SqlTypeFamily.NUMERIC, SqlTypeFamily.NUMERIC)),
     SqlFunctionCategory.NUMERIC)
 
+  val SHIFT_LEFT = new SqlFunction(
+    "SHIFT_LEFT",
+    SqlKind.OTHER_FUNCTION,
+    ReturnTypes.ARG0,
+    null,
+    OperandTypes.family(SqlTypeFamily.NUMERIC, SqlTypeFamily.INTEGER),
+    SqlFunctionCategory.NUMERIC
+  )
+
+  val SHIFT_RIGHT = new SqlFunction(
+    "SHIFT_RIGHT",
+    SqlKind.OTHER_FUNCTION,
+    ReturnTypes.ARG0,
+    null,
+    OperandTypes.family(SqlTypeFamily.NUMERIC, SqlTypeFamily.INTEGER),
```

Comment:
I check the type defined in calcite, `SqlTypeFamily.INTEGER` is 
 `SqlTypeName.INT_TYPES` contains `(TINYINT, SMALLINT, INTEGER, BIGINT)`, So we should change `SqlTypeFamily.NUMERIC` to `SqlTypeFamily.INTEGER` .